### PR TITLE
Downloaded Project opens directly in a Dev Container when user uses Docker

### PR DESCRIPTION
**Pull request by valentinavallalta**
Made changes so that the downloaded opens directly in a dev container if the user has Docker downloaded and they chose to use docker in the docker prompt. In all other senarions, the new project will open in a new window but not in a dev container

### CODE REVIEW

File: src/commands/dockersupport/localDockerSupport.ts [hunk 1]
```diff
@@ -1,8 +1,3 @@
-/*---------------------------------------------------------------------------------------------
```
Comment (refortie) [inline src/commands/dockersupport/localDockerSupport.ts]: Looks like you accidentally removed the Copywrite comment

File: src/downloadAzureProject/setupProjectFolder.ts [hunk 2]
```diff
@@ -31,6 +31,7 @@ export async function setupProjectFolder(uri: vscode.Uri, vsCodeFilePathUri: vsc
     await setupProjectFolderParsed(resourceId, language, vsCodeFilePathUri, context, node, devContainerName);
 }
 
+//call this directly if you are staring from VSCode extension
```
Comment (refortie) [inline src/downloadAzureProject/setupProjectFolder.ts]: Don't think you need this comment necessarily

**Review by refortie — APPROVED** [review]

File: src/commands/dockersupport/localDockerSupport.ts (hunk 1 above)
Comment (valentinavallalta) [reply src/commands/dockersupport/localDockerSupport.ts]: just added it back in!

File: src/downloadAzureProject/setupProjectFolder.ts (hunk 2 above)
Comment (valentinavallalta) [reply src/downloadAzureProject/setupProjectFolder.ts]: Ok, I agree

**Pull request merged**